web-frontend/src/pages/CreateAccount/StepOne.tsx

```diff
@@ -0,0 +1,29 @@
+import { Input } from '@massalabs/react-ui-kit/src/components/Input/Input';
+import { Stepper } from '@massalabs/react-ui-kit/src/components/Stepper/Stepper';
+import { Button } from '@massalabs/react-ui-kit/src/components/Button/Button';
+import { FiArrowRight } from 'react-icons/fi';
+import LandingPage from '../../layouts/LandingPage/LandingPage';
+
+export default function StepOne() {
+  return (
+    <LandingPage>
+      <div
+        className={`flex flex-col justify-center items-center align-center
+                      h-screen`}
+      >
```

Comment:
why prettier let this div on 4 lines?

```suggestion
      <div className={`flex flex-col justify-center items-center align-center h-screen`}>
```

Reply:
i have no idea I'll make the change :/